src/libraries/Microsoft.PowerFx.Core/Texl/Builtins/Error.cs

```diff
@@ -83,7 +83,7 @@ public override bool CheckInvocation(TexlNode[] args, DType[] argTypes, IErrorCo
             }
 
             var argumentKindType = names.First(tn => tn.Name == requiredKindField.Name).Type;
-            if (!argumentKindType.CoercesTo(requiredKindField.Type))
+            if (argumentKindType.Kind != requiredKindField.Type.Kind)
```

Comment:
>if (argumentKindType.Kind != requiredKindField.Type.Kind) [](http://example.com/codeflow?start=12&length=57)

what's the significance of this change?

Reply:
Currently we allow people to write something like `Error({Kind: "a string"})`. This is wrong - the value of Kind should only be a member of the EnumKind enumeration (which is numeric).